src/ext-test/java/org/opentripplanner/ext/geocoder/LuceneIndexTest.java

```diff
@@ -295,9 +302,21 @@ void agenciesAndFeedPublisher() {
       assertEquals(List.of(StopClusterMapper.toAgency(BVG)), cluster.primary().agencies());
       assertEquals("A Publisher", cluster.primary().feedPublisher().name());
     }
+
+    @ParameterizedTest
+    @ValueSource(
+      strings = { "Meridian Ave N & N 148th", "Meridian Ave N & N 148", "Meridian Ave N N 148" }
```

Comment:
Just curious, is this supposed to work with "Meridian & N 148"?

Reply:
I had to check this, but the answer is yes!

I wrote a few more test cases to make sure the it's really the case.